client/src/components/ActionMenu/ActionMenu.js

```diff
@@ -53,6 +53,13 @@ class ActionMenu extends PureComponent {
 
 ActionMenu.defaultProps = {
   className: '',
+  dropdownToggleClassName: [
```

Comment:
Last thing is we should declare the proptype for this e.g. string or array of strings